manim/mobject/types/vectorized_mobject.py

```diff
@@ -101,7 +101,7 @@ def __init__(
         self.shade_in_3d = shade_in_3d
         self.tolerance_for_point_equality = tolerance_for_point_equality
         self.n_points_per_cubic_curve = n_points_per_cubic_curve
-        Mobject.__init__(self, **kwargs)
```

Comment:
Why was this changed? `Mobject.__init__` can still receive keyword arguments, just not arbitrary ones

Reply:
Wait, I'm confused -- so then **kwargs parameter shouldn't be removed from `Mobject.__init__`? The reason I changed this here is because tests would fail if kwargs was removed from mobject.py and not here.

Reply:
If `**kwargs` aren't forwarded from `VMobject` to `Mobject`, things like `color` won't be properly set. `Mobject.__init__` can still *receive* `**kwargs`, that is `**kwargs` from child classes should still be forwarded up the inheritance chain, but `Mobject.__init__`'s signature should have no `**kwargs` because that would mean it could accept *any* keyword argument, such as `dfkjjvbdfkjvdfjkbnfkj=0`

Reply:
I think it we should have **kwargs here at the function call for Mobject.__init__  -- after all **kwargs could contain things like color, name, or dim -- but remove it from the function signature of Mobject.__init__

Some context:

Normally we should always know which kwargs to accept at every level in the derivation tree. If they are not expplicitly given in the function signature but as part of a kwargs dictionary, we should consume them with kwargs.pop and then pass the rest of the kwargs up to the parent until none are left.

When we removed the CONFIG dictionaries and changed everything to be explicit kwargs, we noticed that stray arguments were being passed around at all levels in the code and in the docs ... and it was near impossible to clean it up all at once. For this reason I added the **kwargs to the Container class and there added the logger.debug message about extra kwargs received --  all this on order to allow us to clean up these stray keyword arguments one at a time without causing a crash.